src/components/Checkout/BasketItem.vue

```diff
@@ -147,3 +186,8 @@ export default {
 	},
 };
 </script>
+<style>
+.choose-borrower {
```

Comment:
I think this can be replaced with `tw-cursor-pointer`...

Reply:
done